recipe-client-addon/lib/RecipeRunner.jsm

```diff
@@ -174,13 +199,17 @@ this.RecipeRunner = {
 
       try {
         await manager.runAsyncCallback("postExecution");
+        Uptake.reportAction(actionName, Uptake.ACTION_SUCCESS);
       } catch (err) {
         log.info(`Could not run post-execution hook for ${actionName}:`, err.message);
+        Uptake.reportAction(actionName, Uptake.ACTION_POST_EXECUTION_ERROR);
       }
     }
 
     // Nuke sandboxes
     Object.values(actionSandboxManagers).forEach(manager => manager.removeHold("recipeRunner"));
+
+    Uptake.reportRunner(Uptake.RUNNER_SUCCESS);
```

Comment:
This seems slightly surprising in that it reports success even when there were errors in parts of the execution. Is that intentional?